Add missing meta descriptions (#488)
mongodb/docs-golang: source/usage-examples/updateMany.txt

Update Multiple Documents
=========================

.. meta::

>    :description: Learn how to update multiple documents in a collection using the `UpdateMany()` method in Go, with an example that adjusts prices in a specific market.

Reviewer: ```suggestion
   :description: Learn how to update multiple documents in a collection using the updateMany() method in the MongoDB Go Driver.
```
